pom.xml

```diff
@@ -987,10 +987,6 @@
             <groupId>org.slf4j</groupId>
             <artifactId>jcl-over-slf4j</artifactId>
         </dependency>
-        <dependency>
-            <groupId>ch.qos.logback</groupId>
-            <artifactId>logback-classic</artifactId>
-        </dependency>
```

Comment:
@RaigorJiang What do you think about removing the test dependency of logback globally? Or keeping it?

Reply:
@linghengqian  I'm in favor of removing it from the global, and temporarily disabling the logging rule, since it now binds to logback, and users don't know how to configure the rule.

#23856 provides a good idea, but before we implement the logging rule well, it is better not to let it affect users.

Reply:
So far, the current PR seems to be ready for the next review. In the current PR, Shardingsphere Proxy uses Logback with runtime scope, while ShardingSphere JDBC only provides Logback with provided scope.